frontend/src/components/Dashboard.js

```
@@ -0,0 +1,49 @@
+
+  const handleAvatarFetch = async () => {
+    const response = await fetch(`http://localhost:5001/fetch-avatar?url=${avatarUrl}`);
+    // Handle response...
+  };
+
+  return (
+    <div className="dashboard">
+      <div id="welcome"></div>
+      <div className="profile-section">
+        {profile && (
+          <div dangerouslySetInnerHTML={{ __html: profile.bio }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)